index.js

```diff
@@ -41,7 +48,15 @@ function getAllMovieTitles() {}
  *  getHighestMetascore(movies);
  *  //> 96
  */
-function getHighestMetascore() {}
+function getHighestMetascore(movies) {
+  let metaMovie = 0
+  for(let i = 0; i < movies.length; i++){
+  if(movies[i].metascore > metaMovie){
+ metaMovie = movies[i].metascore
+      }
+  }
+return parseInt(metaMovie);
```

Comment:
Nice work!  Again when I read `metaMovie` I don't know what that means right away.  `highestMeta` lets me know that we would be looking for a 'meta' value.  Also watch our indentation on line 55 - good indentation will help us see where our code blocks start and stop.